### PR TITLE
OT-1702 Add model selector

### PR DESCRIPTION
**Pull request by amilovsky**

### CODE REVIEW

File: keentools_facebuilder/interface/panels.py [hunk 1]
```diff
@@ -535,6 +535,8 @@ def draw(self, context):
         row = box.row()
         row.prop(head, 'model_scale')
 
+        box.prop(head, 'model_level')
```
Comment (SergeyKrivohatskiy) [inline keentools_facebuilder/interface/panels.py:538]: Not sure about the name. It's not level only, maybe there will be other models (e.g. DAZ model)

File: keentools_facebuilder/settings.py [hunk 2]
```diff
@@ -506,6 +522,20 @@ def uv_items_callback(self, context):
     return res
 
 
+def model_level_callback(self, context):
```
Comment (SergeyKrivohatskiy) [inline keentools_facebuilder/settings.py:525]: What is this code for? Do not use names as constants anywhere

Comment (SergeyKrivohatskiy) [reply keentools_facebuilder/settings.py:525]: Use fb.model_names

**Review by SergeyKrivohatskiy — CHANGES_REQUESTED** [review]

File: keentools_facebuilder/settings.py (hunk 2 above)
Comment (amilovsky) [reply keentools_facebuilder/settings.py:525]: model_level_callback generates enum values from fb.model_names
Hardcoded dict is used for additional tooltips and icons if there are known names detected via fb.model_names. In other case there will be fb.mode_names without tooltips and custom icons.

Comment (SergeyKrivohatskiy) [reply keentools_facebuilder/settings.py:525]: Remove Hardcoded dict. If we need additional tooltips we can add them in fb.model_tooltips

**Review by SergeyKrivohatskiy — APPROVED** [review]

**Pull request merged**